styles/style.css

```diff
@@ -62,22 +62,84 @@ body {
 	background: #ce5f31;
 }
 
+
 /* Jumbotron */
 .jumbotron {
 	border-radius: 0;
+	background-image: url(../images/header-bike.jpg)
+}
+.box_container{
+	display: flex;
+align-items: center;
+flex-direction: row;
+justify-content: space-between;
 }
+.box{
+	
+	width:100px;
+    height:100px;
+	background:lightgrey;
+	border: 1px solid black;
+	
+
+	
+}
+.box p {
+padding: 10px;
+display: flex;
+justify-content: center;
+align-content: center;
+text-align: center;
+
+font-size: 20px;
 
+
+}
+
+	
 /* Headings */
 .heading-underline {
 	position: relative;
 	margin-bottom: 2em;
+	margin-top:2em;
 	padding-bottom: 0.5em;
 	border-bottom: 1px solid #dccdc6;
 	font-size: 1rem;
 	font-weight: 600;
 	text-transform: uppercase;
 }
 
+.article-container{
+display: flex;
+flex-wrap: wrap;
+flex-direction: row;
+justify-content: space-between;
```

Comment:
Instead of laying the 2 boxes out with `space-between` here, try laying them out using `row` and `col-` classes directly on the elements. You will need to think of a trick to avoid the boxes touching 🙂